bot/core/bot.py

```diff
@@ -1,20 +1,61 @@
 from datetime import datetime
+from itertools import cycle
+import os
 
-from discord import Color, Embed
+import asyncpg
+from discord import Color, Embed, Game
+from discord.ext import tasks
 from discord.ext.commands import Bot as Base_Bot
 
 from bot import config
 
 
 class Bot(Base_Bot):
-    def __init__(self, extensions: list, *args, **kwargs) -> None:
+    """Subclassed Hotwired bot."""
+
+    def __init__(self, extensions: list, prefix: str, *args, **kwargs) -> None:
+        """Initialize the hotwired bot."""
         super().__init__(*args, **kwargs)
         self.extension_list = extensions
-        self.initial_call = True
+        self.first_on_ready = True
+        self.status = cycle(
+            [
+                (
+                    "😁Working At The Codin' Hole! Join me at "
+                    "https://discord.gg/aYF76yY"
+                ),
+                (
+                    "▶Check out My Creator's Youtube channel : "
+                    "https://www.youtube.com/channel/UC3S4lcSvaSIiT3uSRSi7uCQ/"
+                ),
+                f"Ping me using {prefix}help",
+                (
+                    "Official Instagram of My Creator ❌ "
+                    "https://instagram.com/the.codin.hole/"
+                ),
+                (
+                    "Ready To Work and Get Worked! My Github 🔆 "
+                    "https://github.com/janaSunrise"
+                ),
+            ]
+        )
+
+        self.pool = None
+        self.log_channel = None
 
     async def on_ready(self) -> None:
-        if self.initial_call:
-            self.initial_call = False
+        """Initialize some stuff once the bot is ready."""
+        if self.first_on_ready:
+            self.first_on_ready = False
+
+            self.pool = await asyncpg.create_pool(
+                database=os.getenv("DATABASE_NAME", "hotwired"),
+                host="127.0.0.1", min_size=int(os.getenv("POOL_MIN", "20")),
+                max_size=int(os.getenv("POOL_MAX", "100")),
+                user=os.getenv("DATABASE_USER"),
+                password=os.getenv("DATABASE_PASSWORD"),
+            )
```

Comment:
The database was removed since we're going to fully rely on the [API](https://github.com/The-Codin-Hole/HotWired-API) which will use its database, that way it will be easier to get database results from our [web](https://github.com/The-Codin-Hole/Hotwired-web) with the API acting as a middle point. 

This database pool was removed for a reason and it shouldn't be re-implemented for now.

Reply:
I'm not sure it's the best solution, since it'll add processing time

Reply:
While it will bring additional time, it will be a better option since that way we can easily access the database from anywhere using this API instead of having to rely on internal database on the server this bot will run on.

Reply:
`asyncpg` uses PostgreSQL, and you can easily setup a web interface
For example, Jana can tell you he got access to the database on my server

Reply:
You can also access it with another pool from your web interface.
And have a third one dedicated to the database

Reply:
And why would a program need to access the database from somewhere else ? If both webserver and bot run on the machine, what else could need an access ?

Reply:
The thing is we are not entirely certain that we will run on the same server and if we won't it would be complicated to access this database which would live on a separate server and it would only pose the delay to access this data on the webpage instead of here

Reply:
There's another thing to consider : what part will access the database most often : the webpage or the bot ?
Because this program should be the one with direct access to the database

Reply:
Alright that's a valid point, I suppose we can keep it here than but keep the newer version of this as it was before the db was removed
https://github.com/The-Codin-Hole/HotWired-Bot/blob/1e504b5abf5cc516c7cb88b1bf0ad5778a42b691/bot/core/bot.py#L10-L17
https://github.com/The-Codin-Hole/HotWired-Bot/blob/1e504b5abf5cc516c7cb88b1bf0ad5778a42b691/bot/core/bot.py#L31
```

Reply:
I'll just move the DATABASE variable to the config file